### DISCUSSION TITLE
view/title icon for view action doesn't persist for a webview view

### DISCUSSION THREAD

**Discussion by amitjoshi438** [microsoft/vscode-discussions#638] [Extension Development QnA] [answered]
Repro Steps:
1. Create a webview view using webviewviewprovider.
2. Add a view/title icon "clear all" in the webview title using package.json
3. Load extension.
4. Icon will be only visible when mouse is hovered over title of webview.

**Comment by amitjoshi438** [comment]
@digitarald can you please help?

**Comment by isidorn** [comment, marked as answer]
This is the design decision of views. Currently extension can not control this visibility.
Users can control this behaviour via `"workbench.view.alwaysShowHeaderActions": true`
